Library/Homebrew/cmd/pull.rb

```diff
@@ -211,26 +208,69 @@ def pull
         changed_formulae.each do |f|
           ohai "Installing #{f.full_name}"
           install = f.installed? ? "upgrade" : "install"
-          safe_system "brew", install, "--debug", f.full_name
+          safe_system HOMEBREW_BREW_FILE, install, "--debug", f.full_name
         end
       end
     end
 
-    bintray_fetch_formulae.each do |f|
-      max_retries = 8
+    # Verify bintray publishing after all patches have been applied
+    # Do it in external process to work with fresh formula definitions
+    system HOMEBREW_BREW_FILE, "_internal", "verify-bintray-publish", *bintray_fetch_formulae
```

Comment:
I don't mind this calling out but just make it a `dev-cmd` and skip the `_internal` stuff.